tests/integration/test_logcollector/test_location_wildcards/test_location_wildcards.py

```diff
@@ -0,0 +1,167 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
```

Comment:
```suggestion
copyright: Copyright (C) 2015-2023, Wazuh Inc.
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3906/commits/35f4ac6330db8b957231daa9f2131cf30eeaab0c